test/src/test/java/org/corfudb/infrastructure/LayoutBuilderTest.java

```diff
@@ -23,6 +25,37 @@
     final int SERVER_ENTRY_3 = 3;
     final int SERVER_ENTRY_4 = 4;
 
+    @Test
+    public void emptyLayoutBuilder() {
```

Comment:
This could use a better description or alternatively comments. emptyLayoutBuilderWithSingle might work.

Reply:
Done.